lib/view_component/collection.rb

```diff
@@ -0,0 +1,54 @@
+
+# frozen_string_literal: true
+
+module ViewComponent
+  class Collection
+    def initialize(component, object = nil, options)
```

Comment:
I forget all the rules of public/private in Ruby but can we make this constructor private? You should only interact with it via the class method.

Reply:
Yup! This constructor is now private.